### ISSUE TITLE
Errors compiling on armv7

### ISSUE THREAD

**Issue by berga** [Expaso/hassos-addon-timescaledb#7]
Following issue Expaso/hassos-addons#5
I've tried to compile this on armv7 throws some errors on GO tools and alpine package dependencies

```ERROR: unsatisfiable constraints:
  openexr-2.5.3-r0:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libHalf-2_5.so.25]
               gdal-3.1.3-r1[so:libIex-2_5.so.25]
               gdal-3.1.3-r1[so:libIlmImf-2_5.so.25]
  cfitsio-3.49-r0:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libcfitsio.so.9]
  charls-2.1.0-r1:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libcharls.so.2]
  libdap-3.20.6-r0:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libdap.so.25]
               gdal-3.1.3-r1[so:libdapclient.so.6]
  freexl-1.0.6-r0:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libfreexl.so.1]
  hdf5-1.10.6-r1:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libhdf5.so.103]
  json-c-0.15-r0:
    masked in: @edge
    satisfies: world[json-c]
               postgis-3.0.2-r0[so:libjson-c.so.5]
               gdal-3.1.3-r1[so:libjson-c.so.5]
  libkml-1.3.0-r5:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libkmlbase.so.1]
               gdal-3.1.3-r1[so:libkmldom.so.1]
               gdal-3.1.3-r1[so:libkmlengine.so.1]
  poppler-20.10.0-r0:
    masked in: @edge
    satisfies: gdal-3.1.3-r1[so:libpoppler.so.103]
  proj-datumgrid-7.1.0-r0:
    masked in: @edge
    satisfies: proj-7.1.0-r0[proj-datumgrid]
  proj-7.1.0-r0:
    masked in: @edge
    satisfies: world[proj]
               postgis-3.0.2-r0[so:libproj.so.20]
               gdal-3.1.3-r1[so:libproj.so.20]
  gdal-3.1.3-r1:
    masked in: @edge
    satisfies: world[gdal]
               postgis-3.0.2-r0[so:libgdal.so.27]
  postgis-3.0.2-r0:
    masked in: @edge
    satisfies: world[postgis]
```

**Comment by Expaso** [comment]
Nice one! Let's see if this passes CI. 
We have to test the upgrade path for existing users though, because of the different postgis package.
Thanks for the PR!

**Comment by Expaso** [comment]
Looks like we could also enable x86..  Although I don't know if this is used anywhere in home-assistant world.

**Comment by Expaso** [comment]
Can you try using docker image 1.1.5 manually? It's a prelease now.

**Comment by berga** [comment]
Hi,

I'm sorry for the delay, I'm running the latest version since last commit and it's going well:

```
Add-on version: 1.1.5
 You are running the latest version of this add-on.
parse error: Expected string key before ':' at line 1, column 4
[17:37:28] ERROR: Unknown HTTP error occured
 System:   (armv7 / raspberrypi4)
 Home Assistant Core: 0.116.4
 Home Assistant Supervisor: 247
```

**Comment by Expaso** [comment]
Allright! I will upgrade the instance to support armhf/i386 as well..

**Comment by Expaso** [comment]
Released v1.1.6, which also added armhf and i386.

Thanks @berga for your help!

**Comment by boesing** [comment]
The addon works for 32-bit but the postgis extension doesn't.

I'm getting the following error when I want to create the extension:
```
ERROR: could not load library "/usr/lib/postgresql/postgis-3.so": Error relocating /usr/lib/postgresql/postgis-3.so: __time64: symbol not found
```

Is that 64-bit related?

**Comment by Expaso** [comment]
Hmm I suppose it is indeed. See if we can get this fixed for you. Whats is your current setup? Like device&architechture?

**Comment by boesing** [comment]
@Expaso I was on Raspberry Pi 4 with HassOS 32-bit version.
I've recently migrated to HassOS 64-bit due to a migration to full SSD.

So now I am on Raspberry Pi 4 with NVMe SSD with HassOS 64-bit 5.0-beta and this plugin works as expected. 👍 
But there might be others out there which might want to use Postgis.

**Comment by Expaso** [comment]
Should be fixed in the upcoming 2.0.0 release.

Caution! Running PostgreSQL on non 64-Bit architectures is no longer supported due to the lack of resources (mem <2GB).
Since the 64-bit versions of Hassos are now the recommended versions, support for non 64-bit architectures will be removed in future versions of this addon.

**Comment by boesing** [comment]
I've already upgrade to x64 so no problem here. Thanks for the feedback anyways!
